generators/client/templates/angular/webpack/_webpack.dev.js

```diff
@@ -19,9 +49,13 @@ module.exports = webpackMerge(commonConfig({env: ENV}), {
         new BrowserSyncPlugin({
             host: 'localhost',
             port: 9000,
-            proxy: 'http://localhost:<%= serverPort %>'
+            proxy: 'http://localhost:9002'
```

Comment:
Why don't we keep `<%= serverPort %>`? This is a must for gateways and also useful for #4844